_posts/2015/2016-07-20-fullstackcon-2016-retrospective.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages goes FullStack JavaScript conference"
+date: "2016-07-20 12:00:00"
+image: blog-header/javascript.png
```

Comment:
I love the JS header image BUT I'm not sure, if we're allowed to use it. Do you know?
Where did you get it from? If unsure, you can try find a header image at pexels or pixabay.